javascript.md

```diff
@@ -344,14 +344,6 @@ Try/Catch should always be used when accessing deeply nested objects, when you c
 
 ### 9.1 jQuery
 
-#### Use in all QUnit tests
```

Comment:
I think we said we DO want to write JQ in Qunit tests, right? But the issue is confusing me.  @jduuuubs, do you remember?